.env.example

```diff
@@ -23,8 +23,9 @@ OPENAPI_DOCS_AUTH_CLIENT_ID=${OPENAPI_DOCS_AUTH_CLIENT_ID}
 # S3 Credentials
 # For more information on populating these please see
 # https://docs.digitalocean.com/products/spaces/reference/s3-sdk-examples/
-S3_ENDPOINT=${S3_ENDPOINT}
 S3_ACCESS_KEY_ID=${S3_ACCESS_KEY_ID}
 S3_ACCESS_SECRET=${S3_ACCESS_SECRET}
-S3_UNPROCESSED_BUCKET=${S3_UNPROCESSED_BUCKET}
-S3_BATCH_UPLOADS_BUCKET=${S3_BATCH_UPLOADS_BUCKET}
+S3_BUCKET=${S3_BUCKET}
+S3_ENDPOINT=${S3_ENDPOINT}
+S3_PATH_STYLE=$(( true || false ))
```

Comment:
This format doesn't work with `envsubst`, right? (E.g. if someone runs `envsubst .env.example > .env` I don't think they would be able to populate `S3_PATH_STYLE` without additional commands)

Can we maintain the `${S3_PATH_STYLE}` approach but include legitimate values in a comment?

Reply:
Ahh, is that the motivation for this format? I wasn't sure why they were formatted using shell variables rather than example values. ...are we actually doing that anywhere? Should we document that? I'm not sure I understand the use case, tbh!

Reply:
It's just an accessibility thing that I've found to be best practice.  If you have a strong reason against it I'm happy to hear it!  I think signaling values can easily be done through comments (and prefer that anyway since the comments live on once the value is replaced)

Reply:
Thanks for explaining!